tests/conftest.py

```diff
@@ -15,6 +16,12 @@ def account_manager(account_json=None):
     return account_factory.get_account(account_json)
 
 
+@pytest.fixture
+def bucket_manager():
+    bucket_factory = BucketOperation()
+    return bucket_factory
+
```

Comment:
@sagihirshfeld For now I have added clean up task as todo since this PR only checks commands are working correctly on latest build. I will send clean up code in upcoming PRs